rpc/transaction.go

```diff
@@ -88,6 +88,7 @@ type TxnFinalityStatus uint8
 const (
 	TxnAcceptedOnL1 TxnFinalityStatus = iota + 1
 	TxnAcceptedOnL2
+	TxNotRECEIVED
```

Comment:
We shouldn't add this here. It isn't part of the [RPC spec](https://github.com/starkware-libs/starknet-specs/blob/6fadca1f1abf48523c0aa799a10061f40f973468/api/starknet_api_openrpc.json#L2532-L2540).

Reply:
removed